docs/azure/services/Azure_Functions.md

```diff
@@ -0,0 +1,109 @@
+
+### Encryption in Transit
+Azure Functions encrypts all the incoming data using TLS 1.2 by default. However, the service does allow for manual downgrading. When reviewing the configuration, users must be ensure that TLS 1.2 is enforced for all communication to the deployed function.
+
+Furthermore, clients connecting to the function can opt to use either HTTP or HTTPS. Where possible, any HTTP traffic should be redirected to HTTPS. Reviewing whether the latest version of TLS and HTTPS only traffic has been enforced, can be done either in the `TLS/SSL` settings blade or via the following CLI command:
+```CLI
+az functionapp show --name MyFunctionApp --resource-group MyResourceGroup --query “httpsOnly”
+az functionapp show --name MyFunctionApp --resource-group MyResourceGroup --query "minTlsVersion"
+```
+Azure Functions have an FTP endpoint that is enabled by default. If not used, this endpoint should be disabled to restrict the possible ways the function app can be interacted with. If required as part of business operation, deployments should be forced to use FTPS. To check the FTP configuration, the following command could be run:
+```CLI
+az functionapp show --name AzFunTestBl --resource-group AzFun --query siteConfig."ftpsState"
+```
+
+### Function Access Keys
+Unless a HTTP-triggered function is set to “anonymous”, two authorisation levels configured on creation determine whether the function app is using a “function-level” or “admin-level” access key. 
+
+Function-level access key can authorise against a specific function, whilst admin-level uses a “master-key” for authentication against all functions in the Function App. Each Function App has a master-key on creation which can’t be removed. Additionally, it also provides administrative access to the runtime’s APIs. Because of the authorisation scope that the master key provides, users should be careful with how it is handled and distributed as part of their secrets management process.
+
+Additionally, a “system-key” is used for webhook authentication required by different extensions used within a Function App. This system-key will be created by extension which will also determine whether it’s a host-level or function-level key.
+
+As with other resources, relying on shared keys to access a resource is not an ideal scenario in terms of secrets management across an estate. Under the Authentication blade, it is possible to set up an authentication provider such as Azure AD which makes use of the tenant's authentication capabilities to restrict access to function endpoints. Further information can be found [here](https://learn.microsoft.com/en-us/azure/app-service/overview-authentication-authorization).
```

Comment:
What about the usage of API Management to authenticate requests to the function app? Since that's also a suggested and valid way of handling auth to your functions.